plugin/pxr/maya/lib/usdMaya/testenv/testUsdExportPointInstancer.py

```diff
@@ -340,6 +341,10 @@ def testMashInstancePaths(self):
         self._TestInstancePaths("MASH1_Instancer")
 
     def testMashVisibility(self):
+
+        # assertCountEqual in python 3 is equivalent to assertItemsEqual
+        if sys.version_info[0] >= 3:
+            self.assertItemsEqual = self.assertCountEqual
```

Comment:
Python 3 no longer has assertItemsEqual and instead this function has been renamed to assertCountEqual.

added `self.assertItemsEqual = self.assertCountEqual` for compatibility between Python 2 and Python 3